barcode_action/wizard/barcode_action.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class BarcodeAction(models.TransientModel):
+    _name = "barcode.action"
+    _inherit = "barcodes.barcode_events_mixin"
```

Comment:
Please add `_description` for fix ci/runbot.

Reply:
:+1: